<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWIndow</class>
 <widget class="QMainWindow" name="MainWIndow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>592</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>车辆预约</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>671</width>
      <height>1031</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>669</width>
       <height>1029</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>641</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget {border:1px solid rgb(211,211,211); border-radius:5px}</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_2{border:none; 
border-bottom:1.2px solid rgb(211,211,211)}</string>
       </property>
      </widget>
      <widget class="QLabel" name="name">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>预约人姓名：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>人事编号：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit{border:none; 
border-bottom:1.2px solid rgb(211,211,211)}</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>所属校区：</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>主校区</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>110</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>同济校区</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>641</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_2 {border:1px solid rgb(211,211,211); border-radius:5px}</string>
      </property>
      <widget class="QLabel" name="name_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>进校方式：</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>步行</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>驾车</string>
       </property>
      </widget>
      <widget class="QLabel" name="name_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>车牌号码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#lineEdit_3{border:none; 
border-bottom:1.2px solid rgb(211,211,211)}</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>250</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="name_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>来访人员：</string>
       </property>
      </widget>
      <widget class="QLabel" name="name_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>来访时间：</string>
       </property>
      </widget>
      <widget class="QLabel" name="name_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>访问事由：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>290</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">  ui-&gt;comboBox-&gt;addItem(icon,&quot;四川&quot;);
 ui-&gt;comboBox-&gt;addItem(icon,&quot;湖南&quot;);
    ui-&gt;comboBox-&gt;addItem(icon,&quot;湖北&quot;);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AddButton">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>105</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#AddButton{border:none; 
image: url(:/new/prefix1/UIImages/plus.png);}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QTableWidget" name="table">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>341</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:gray</string>
       </property>
       <property name="text">
        <string>(最多添加4人)</string>
       </property>
      </widget>
      <zorder>name_2</zorder>
      <zorder>checkBox_3</zorder>
      <zorder>checkBox_5</zorder>
      <zorder>name_3</zorder>
      <zorder>lineEdit_3</zorder>
      <zorder>dateEdit</zorder>
      <zorder>name_4</zorder>
      <zorder>name_5</zorder>
      <zorder>name_6</zorder>
      <zorder>comboBox</zorder>
      <zorder>table</zorder>
      <zorder>AddButton</zorder>
      <zorder>label_4</zorder>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>830</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_2 {border:1px solid rgb(211,211,211); border-radius:12px;background-color: rgb(0, 114, 169);
}

</string>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>790</y>
        <width>16</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>641</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_4 {
background-image: url(:/new/prefix1/UIImages/hust1.png);
border-radius:5px;
}
</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>730</y>
        <width>641</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_3 {border:1px solid rgb(211,211,211); border-radius:5px}</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>入校短信通知</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>0</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox::indicator:unchecked {
	image: url(:/new/prefix1/UIImages/switch-OFF.png);
}
QCheckBox::indicator:checked {
  image: url(:/new/prefix1/UIImages/switch-ON.png);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>890</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_3 {border:1px solid rgb(211,211,211); border-radius:12px;background-color: rgb(0, 114, 169);
}

</string>
      </property>
      <property name="text">
       <string>我的预约</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>790</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>我已认真阅读并同意以上协议</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>950</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_4{border:1px solid rgb(211,211,211); border-radius:12px;background-color: rgb(0, 114, 169);
}

</string>
      </property>
      <property name="text">
       <string>退出预约</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
